src/main/resources/org/boozallen/plugins/jte/init/primitives/injectors/StepWrapper.groovy

```diff
@@ -86,7 +86,14 @@ class StepWrapper extends TemplatePrimitive implements Serializable, Cloneable{
      */
     private HookContext hookContext
```

Comment:
does the hookContext need to stay private?

Reply:
i removed the private modifier because for some reason (**probably CPS related**), when the field was private the getters were returning `null` even if the field was set.

Reply:
so to answer your actual question - yes - the hookContext should stay private